content/en/docs/releases/next-release-preview/index.md

```diff
@@ -69,3 +69,10 @@ In this example, even though each trigger has its own artifact(s) defined, when
 - Replace deprecated Azure SDK `com.microsoft.azure:azure` with the new Azure SDK `com.azure.resourcemanager:azure-resourcemanager`
 - Add a new `AzureManagedImageCachingAgent` which is caching managed images from the specified region and resource group
 - Update `AzureVMImageLookupController` to return the newly cached managed images when `managedImages` query parameter is set to `true`
+
+### Addition of includeEvents parameter
+`includeEvents` has been added as a configurable parameter in orca that enables users to fetch [Kubernetes Events](https://kubernetes.io/docs/reference/kubernetes-api/cluster-resources/event-v1/) from clouddriver. This parameter defaults to `false`.
+
+![includeEvents](includeEvents.png)
```

Comment:
I'd still prefer we leave out the bit about setting up a pipeline parameter for this.  People build pipelines in lots of ways, and set fields in pipeline stages in lots of ways.  This doesn't feel like it really adds to the discussion of `includeEvents` specifically.

Reply:
@dbyron-sf Removed the configuration bit from the docs. Please lemme know if this is okay?